core/coreunix/add.go

```diff
@@ -58,7 +190,10 @@ func AddR(n *core.IpfsNode, root string) (key string, err error) {
 	}
 	defer f.Close()
 
-	dagnode, err := addFile(n, f)
+	fileAdder := NewAdder(n.Context(), n, nil)
+	fileAdder.Chunker = "default"
```

Comment:
maybe the the constructor should set the default value of `Chunker` to "default" ? or maybe assume that it being unset means to use the default?